test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -194,12 +194,13 @@ def setUp(self):
         self.eatmem_job2 = \
             '#PBS -joe\n' \
             '#PBS -S /bin/bash\n' \
+            'sync\n' \
```

Comment:
Should we be adding a call to sync to the other self.eatmem_job* sections as well? Should we add a call to sync before/after each of the self.eatmem_script substitutions?

Reply:
There is a need to sync the buffers before trying to fill them up. I can see that it will also apply to the others (eatmem_job1 and eatmem_job3).  I'll make the changes and run the affected tests.